catalog/core/catalog-core-api-impl/src/test/java/ddf/catalog/data/impl/MetacardTypeImplTest.java

```diff
@@ -353,15 +350,6 @@ public void testMediaType() {
     }
 
     @Test
-    public void testMetacardType() {
```

Comment:
They don't need to be refactored; now that the Metacard attributes are in core, the tests for the Core Attributes will be sufficient